async_fuse/src/byte_slice.rs

```diff
@@ -106,18 +118,20 @@ impl<'a> ByteSlice<'a> {
         Ok(ret)
     }
 
-    pub(crate) fn fetch_c_str(&mut self) -> anyhow::Result<&'a CStr> {
+    /// Fetch some bytes as a C-string
+    pub fn fetch_c_str(&mut self) -> anyhow::Result<&'a CStr> {
         let strlen: usize = match memchr::memchr(b'\0', self.data) {
-            Some(nul_pos) => nul_pos + 1,
+            Some(nul_pos) => nul_pos.overflow_add(1),
```

Comment:
`nul_pos` is always less then `self.data.len()`.
So `nul_pos + 1` is impossible to overflow unless `memchr` releases a **wrong** version.
Consider using `wrapping_add`.

Reply:
same above